CMakeLists.txt

```diff
@@ -260,7 +260,9 @@ if(PROJECT_SOURCE_DIR STREQUAL PROJECT_BINARY_DIR)
 endif()
 
 if (NOT CMAKE_BUILD_TYPE)
-    set( CMAKE_BUILD_TYPE Release)
+   set( CMAKE_BUILD_TYPE Release)
+elseif(CMAKE_BUILD_TYPE STREQUAL "PureDebug")
```

Comment:
This only handles one particular case. Any misspelled or non-standard build type will fail. It would be better to warn and switch to one of the standard ones (e.g., Release).